capstone/src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -43,7 +43,13 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String ChannelId = request.getParameter("channelId");
-    String url = URL + ChannelId + "&maxResults=" + NUM_RESULTS + "&key=" + KEY;
+    String videoID = request.getParameter("videoID");
```

Comment:
nit: Let's keep a consistent naming style between `ChannelId` and `videoID` - I would recommend `channelId` and `videoId`

Reply:
addressed in "spacing and address Daniel's nits"